modules/guac/main.tf

```diff
@@ -1,46 +1,48 @@
-    "BrandText"                = "${var.BrandText}"
-    "CloudWatchAgentUrl"       = "${var.CloudWatchAgentUrl}"
-    "DesiredCapacity"          = "${var.DesiredCapacity}"
-    "ForceUpdateToggle"        = "${var.ForceUpdateToggle}"
-    "GuacBaseDN"               = "${var.GuacBaseDN}"
-    "GuacamoleVersion"         = "${var.GuacamoleVersion}"
-    "GuacdVersion"             = "${var.GuacdVersion}"
-    "InstanceType"             = "${var.InstanceType}"
-    "KeyPairName"              = "${var.KeyPairName}"
-    "LdapDN"                   = "${var.LdapDN}"
-    "LdapServer"               = "${var.LdapServer}"
-    "MaxCapacity"              = "${var.MaxCapacity}"
-    "MinCapacity"              = "${var.MinCapacity}"
-    "PrivateSubnetIDs"         = "${join(",", var.PrivateSubnetIDs)}"
-    "PublicSubnetIDs"          = "${join(",", var.PublicSubnetIDs)}"
-    "ScaleDownDesiredCapacity" = "${var.ScaleDownDesiredCapacity}"
-    "ScaleDownSchedule"        = "${var.ScaleDownSchedule}"
-    "ScaleUpSchedule"          = "${var.ScaleUpSchedule}"
-    "SslCertificateName"       = "${var.SslCertificateName}"
-    "SslCertificateService"    = "${var.SslCertificateService}"
-    "URL1"                     = "${var.URL1}"
-    "URL2"                     = "${var.URL2}"
-    "URLText1"                 = "${var.URLText1}"
-    "URLText2"                 = "${var.URLText2}"
-    "UpdateSchedule"           = "${var.UpdateSchedule}"
-    "VPC"                      = "${var.VPC}"
+    "AmiId"                    = var.AmiId
+    "AmiNameSearchString"      = var.AmiNameSearchString
+    "BrandText"                = var.BrandText
+    "CloudWatchAgentUrl"       = var.CloudWatchAgentUrl
+    "DesiredCapacity"          = var.DesiredCapacity
+    "ForceUpdateToggle"        = var.ForceUpdateToggle
+    "GuacBaseDN"               = var.GuacBaseDN
+    "GuacamoleVersion"         = var.GuacamoleVersion
+    "GuacdVersion"             = var.GuacdVersion
+    "InstanceType"             = var.InstanceType
+    "KeyPairName"              = var.KeyPairName
+    "LdapDN"                   = var.LdapDN
+    "LdapServer"               = var.LdapServer
+    "MaxCapacity"              = var.MaxCapacity
+    "MinCapacity"              = var.MinCapacity
+    "PrivateSubnetIDs"         = join(",", var.PrivateSubnetIDs)
+    "PublicSubnetIDs"          = join(",", var.PublicSubnetIDs)
+    "ScaleDownDesiredCapacity" = var.ScaleDownDesiredCapacity
+    "ScaleDownSchedule"        = var.ScaleDownSchedule
+    "ScaleUpSchedule"          = var.ScaleUpSchedule
+    "SslCertificateName"       = var.SslCertificateName
+    "SslCertificateService"    = var.SslCertificateService
+    "URL1"                     = var.URL1
+    "URL2"                     = var.URL2
+    "URLText1"                 = var.URLText1
+    "URLText2"                 = var.URLText2
+    "UpdateSchedule"           = var.UpdateSchedule
+    "VPC"                      = var.Vpc
   }
 
   timeouts {
```

Comment:
might be better to define these inputs as separate vars. if the desire is to set all inputs with the same value you could have a conditional that if a var is set (something like `CFNGloablTimeoutInMinutes`) then set all inputs to that value, else, use the individual values. Alternatively, recommend moving the `"${var.TimeoutInMinutes}m"` string to a local

Reply:
Good point.  I'll change the timeouts use separate variables